doc/reference/reference_lua/box_ctl/demote.rst

```diff
@@ -0,0 +1,28 @@
+.. _box_ctl-demote:
+
+box.ctl.demote()
+=================
+
+..  function:: demote()
+
+    Revoke the leader role of the instance.
+
+    The function is NOP (No operation) on any instance but the
+    :ref:`synchronous transaction queue owner <box_info_synchro>`.
+    If the node is not the owner, the function does nothing and returns immediately.
```

Comment:
The second sentence rephrases the first one. Consider shortening this paragraph.
And maybe even move it to the end of the description. Thus, you'll start from the actual case (_On synchronous transaction queue owner_) and then say how the function works on other instances.
Smth like: "On instances that aren't queue owners, the function does nothing and returns immediately"